### PR TITLE
Remove kotlin-android-extensions

### PR DESCRIPTION
**Pull request by jplianghu**
Remove usage of kotlin-android-extenstions and replace kotlin synthetic views with findViewById()

### CODE REVIEW

File: AppLovin MAX Demo App - Kotlin/app/src/main/java/com/applovin/enterprise/apps/demoapp/ads/DemoMenuActivity.kt [hunk 1]
```diff
@@ -9,19 +9,21 @@ import android.view.View
...
+import android.widget.ListView
 import android.widget.TextView
 import com.applovin.enterprise.apps.demoapp.R
 import com.applovin.enterprise.apps.demoapp.data.main.DemoMenuItem
-import kotlinx.android.synthetic.main.activity_list.*
 
 
 abstract class DemoMenuActivity : AppCompatActivity()
 {
+    protected lateinit var list_view: ListView
```
Comment (santoshbagadi) [inline AppLovin MAX Demo App - Kotlin/app/src/main/java/com/applovin/enterprise/apps/demoapp/ads/DemoMenuActivity.kt:20]: Let's match Java's naming here. 
```suggestion
    protected lateinit var listView: ListView

```

File: AppLovin MAX Demo App - Kotlin/app/src/main/java/com/applovin/enterprise/apps/demoapp/ads/applovin/interstitials/InterstitialZoneActivity.kt [hunk 2]
```diff
@@ -1,16 +1,17 @@
...
 import com.applovin.enterprise.apps.demoapp.ui.BaseAdActivity
 import com.applovin.sdk.*
-import kotlinx.android.synthetic.main.activity_interstitial_manual_loading.*
 
 class InterstitialZoneActivity : BaseAdActivity(),
         AppLovinAdLoadListener, AppLovinAdDisplayListener, AppLovinAdClickListener, AppLovinAdVideoPlaybackListener
 {
     private var currentAd: AppLovinAd? = null
+    private lateinit var showButton: Button
     override fun onCreate(savedInstanceState: Bundle?)
```
Comment (santoshbagadi) [inline AppLovin MAX Demo App - Kotlin/app/src/main/java/com/applovin/enterprise/apps/demoapp/ads/applovin/interstitials/InterstitialZoneActivity.kt:15]:
```suggestion
    private lateinit var showButton: Button

    override fun onCreate(savedInstanceState: Bundle?)
```

File: AppLovin MAX Demo App - Kotlin/app/src/main/java/com/applovin/enterprise/apps/demoapp/ads/applovin/leaders/LeaderProgrammaticActivity.kt [hunk 3]
```diff
@@ -31,9 +31,11 @@ class LeaderProgrammaticActivity : BaseAdActivity(),
         adView.setAdClickListener(this)
 
         adView.id = ViewCompat.generateViewId()
+        val load_button = findViewById<Button>(R.id.load_button)
```
Comment (santoshbagadi) [inline AppLovin MAX Demo App - Kotlin/app/src/main/java/com/applovin/enterprise/apps/demoapp/ads/applovin/leaders/LeaderProgrammaticActivity.kt:34]:
```suggestion
        adView.id = ViewCompat.generateViewId()

        val load_button = findViewById<Button>(R.id.load_button)
```

**Review by santoshbagadi — COMMENTED** [review]
We should use `lowerCamelCase` for variable names everywhere.

File: AppLovin MAX Demo App - Kotlin/app/src/main/java/com/applovin/enterprise/apps/demoapp/ads/DemoMenuActivity.kt (hunk 1 above)
Comment (thomasmso) [reply AppLovin MAX Demo App - Kotlin/app/src/main/java/com/applovin/enterprise/apps/demoapp/ads/DemoMenuActivity.kt:20]: @jplianghu - should look around the codebase on our conventions

Comment (thomasmso) [reply AppLovin MAX Demo App - Kotlin/app/src/main/java/com/applovin/enterprise/apps/demoapp/ads/DemoMenuActivity.kt:20]: For example if it's lowerCamelCase everywhere, and you use snake case, and if everyone starts using their own convention, the code becomes unreadable very fast.

Comment (thomasmso) [reply AppLovin MAX Demo App - Kotlin/app/src/main/java/com/applovin/enterprise/apps/demoapp/ads/DemoMenuActivity.kt:20]: For layouts we don't really have a convention, so you may see snake_case there.

**Comment by jplianghu** [comment]
Changes made for Renaming snake_case view names to lowerCamelCase

File: AppLovin MAX Demo App - Kotlin/app/src/main/java/com/applovin/enterprise/apps/demoapp/ads/DemoMenuActivity.kt [hunk 4]
```diff
@@ -1,27 +1,28 @@
...
+import android.widget.ListView
 import android.widget.TextView
 import com.applovin.enterprise.apps.demoapp.R
 import com.applovin.enterprise.apps.demoapp.data.main.DemoMenuItem
-import kotlinx.android.synthetic.main.activity_list.*
 
 
 abstract class DemoMenuActivity : AppCompatActivity()
 {
+    protected lateinit var listView: ListView
```
Comment (thomasmso) [inline AppLovin MAX Demo App - Kotlin/app/src/main/java/com/applovin/enterprise/apps/demoapp/ads/DemoMenuActivity.kt:19]: We usually have a new line separating field declarations
```suggestion
    protected lateinit var listView: ListView

```

File: AppLovin MAX Demo App - Kotlin/app/src/main/java/com/applovin/enterprise/apps/demoapp/ads/applovin/banners/BannerProgrammaticActivity.kt [hunk 5]
```diff
@@ -33,15 +33,18 @@ class BannerProgrammaticActivity : BaseAdActivity(),
...
 
         adView.id = ViewCompat.generateViewId()
-        load_button.setOnClickListener { adView.loadNextAd() }
+        val loadButton = findViewById<Button>(R.id.load_button)
+        loadButton.setOnClickListener { adView.loadNextAd() }
 
         // Add programmatically created banner into our container
-        banner_programmatic_content_layout.addView(adView, ConstraintLayout.LayoutParams(ViewGroup.LayoutParams.MATCH_PARENT, AppLovinSdkUtils.dpToPx(this, 50)))
+        val bannerProgrammaticContentLayout =
+            findViewById<ConstraintLayout>(R.id.banner_programmatic_content_layout)
```
Comment (thomasmso) [inline AppLovin MAX Demo App - Kotlin/app/src/main/java/com/applovin/enterprise/apps/demoapp/ads/applovin/banners/BannerProgrammaticActivity.kt:41]:
```suggestion
        val bannerProgrammaticContentLayout = findViewById<ConstraintLayout>(R.id.banner_programmatic_content_layout)
```

File: AppLovin MAX Demo App - Kotlin/app/src/main/java/com/applovin/enterprise/apps/demoapp/ads/applovin/banners/BannerProgrammaticActivity.kt [hunk 6]
```diff
@@ -33,15 +33,18 @@ class BannerProgrammaticActivity : BaseAdActivity(),
         adView.setAdClickListener(this)
 
         adView.id = ViewCompat.generateViewId()
-        load_button.setOnClickListener { adView.loadNextAd() }
+        val loadButton = findViewById<Button>(R.id.load_button)
+        loadButton.setOnClickListener { adView.loadNextAd() }
 
         // Add programmatically created banner into our container
-        banner_programmatic_content_layout.addView(adView, ConstraintLayout.LayoutParams(ViewGroup.LayoutParams.MATCH_PARENT, AppLovinSdkUtils.dpToPx(this, 50)))
+        val bannerProgrammaticContentLayout =
```
Comment (thomasmso) [inline AppLovin MAX Demo App - Kotlin/app/src/main/java/com/applovin/enterprise/apps/demoapp/ads/applovin/banners/BannerProgrammaticActivity.kt:40]: Feel we don't need to be this verbose with the naming, e.g. `contentLayout` would be fine

**Comment by jplianghu** [comment]
Want to pushed 2 commits below, please review.
[Improve readability by renaming variable & separating spaces](https://github.com/AppLovin/AppLovin-MAX-SDK-Android/pull/417/commits/0b8cb06ae30ac74045ac991f1e6fe0077ea6374f)
[Improve consistence with Java Demo](https://github.com/AppLovin/AppLovin-MAX-SDK-Android/pull/417/commits/519b5c249017302d5488e3dccbe775e4a0f0af92)

**Comment by thomasmso** [comment]
@jplianghu - Development process is to just make the changes onto your branch, then after the changes are made, type in comments "changes made". There are times we make PRs into PRs but this is not one of them.

File: AppLovin MAX Demo App - Kotlin/app/src/main/java/com/applovin/enterprise/apps/demoapp/ads/applovin/leaders/LeaderLayoutEditorActivity.kt [hunk 7]
```diff
@@ -30,7 +30,6 @@ class LeaderLayoutEditorActivity : BaseAdActivity(),
         adView.setAdClickListener(this)
 
         val loadButton = findViewById<Button>(R.id.load_button)
-
```
Comment (thomasmso) [inline AppLovin MAX Demo App - Kotlin/app/src/main/java/com/applovin/enterprise/apps/demoapp/ads/applovin/leaders/LeaderLayoutEditorActivity.kt:33]: 👍

File: AppLovin MAX Demo App - Kotlin/app/src/main/java/com/applovin/enterprise/apps/demoapp/ads/max/RewardedAdActivity.kt [hunk 8]
```diff
@@ -21,7 +23,7 @@ import java.util.concurrent.TimeUnit
 class RewardedAdActivity : BaseAdActivity(),
         MaxRewardedAdListener, MaxAdRevenueListener {
     private lateinit var rewardedAd: MaxRewardedAd
-    private var retryAttempt = 0.0
+    private var retryAttempt: Int = 0
```
Comment (thomasmso) [inline AppLovin MAX Demo App - Kotlin/app/src/main/java/com/applovin/enterprise/apps/demoapp/ads/max/RewardedAdActivity.kt:26]: We usually use type inference in Kotlin and Swift (less code, more "modern"), so `private var retryAttempt` should be good enough.

File: AppLovin MAX Demo App - Kotlin/app/src/main/java/com/applovin/enterprise/apps/demoapp/ads/max/banner/LayoutEditorBannerAdActivity.kt [hunk 9]
```diff
@@ -20,23 +20,26 @@ import kotlinx.android.synthetic.main.activity_layout_editor_banner_ad.*
  */
 class LayoutEditorBannerAdActivity : BaseAdActivity(),
         MaxAdViewAdListener, MaxAdRevenueListener {
+    private lateinit var adView: MaxAdView
```
Comment (thomasmso) [inline AppLovin MAX Demo App - Kotlin/app/src/main/java/com/applovin/enterprise/apps/demoapp/ads/max/banner/LayoutEditorBannerAdActivity.kt:23]:
```suggestion

        private lateinit var adView: MaxAdView

```

File: AppLovin MAX Demo App - Kotlin/app/src/main/java/com/applovin/enterprise/apps/demoapp/ads/max/mrecs/LayoutEditorMrecAdActivity.kt [hunk 10]
```diff
@@ -20,18 +20,20 @@ import kotlinx.android.synthetic.main.activity_layout_editor_mrec_ad.*
  * Created by Andrew Tian on 2020-01-14.
  */
 class LayoutEditorMrecAdActivity : BaseAdActivity(), MaxAdViewAdListener, MaxAdRevenueListener {
+    private lateinit var adView: MaxAdView
```
Comment (thomasmso) [inline AppLovin MAX Demo App - Kotlin/app/src/main/java/com/applovin/enterprise/apps/demoapp/ads/max/mrecs/LayoutEditorMrecAdActivity.kt:23]:
```suggestion

    private lateinit var adView: MaxAdView
```

File: AppLovin MAX Demo App - Kotlin/app/src/main/java/com/applovin/enterprise/apps/demoapp/ads/max/mrecs/ProgrammaticMrecAdActivity.kt [hunk 11]
```diff
@@ -21,56 +21,59 @@ import com.applovin.sdk.AppLovinSdkUtils
  * Created by Andrew Tian on 2020-01-14.
  */
 class ProgrammaticMrecAdActivity : BaseAdActivity(), MaxAdViewAdListener, MaxAdRevenueListener {
-    private var adView: MaxAdView? = null
+    private lateinit var adView: MaxAdView
```
Comment (thomasmso) [inline AppLovin MAX Demo App - Kotlin/app/src/main/java/com/applovin/enterprise/apps/demoapp/ads/max/mrecs/ProgrammaticMrecAdActivity.kt:24]:
```suggestion

    private lateinit var adView: MaxAdView

```

File: AppLovin MAX Demo App - Kotlin/app/src/main/res/layout/activity_leader_programmatic.xml [hunk 12]
```diff
@@ -6,7 +6,7 @@
     android:layout_width="match_parent"
     android:layout_height="match_parent"
     android:id="@+id/leader_programmatic_layout"
-    tools:context=".leaders.LeaderProgrammaticActivity">
+    tools:context="com.applovin.enterprise.apps.demoapp.ads.applovin.leaders.leaders.LeaderProgrammaticActivity">
```
Comment (thomasmso) [inline AppLovin MAX Demo App - Kotlin/app/src/main/res/layout/activity_leader_programmatic.xml:9]: Are you sure this is correct? I see `leaders.leaders`.

**Comment by jplianghu** [comment]
Changes made

**Review by thomasmso — APPROVED** [review]

**Pull request merged**